backend/app/services/embedding.py

```
@@ -208,17 +218,19 @@ def _standard_normalization(self, text: str) -> str:
 
         # Normalize quotes
         text = text.replace('"', '"').replace('"', '"')
-        text = text.replace(''', "'").replace(''', "'")
+        text = text.replace(""", "'").replace(""", "'")
 
         # Normalize common Unicode characters
-        text = text.replace('–', '-').replace('—', '--')
-        text = text.replace('…', '...')
+        text = text.replace("–", "-").replace("—", "--")
```

Ruff (0.13.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)